crengine/src/private/lvfreetypeface.cpp

```diff
@@ -1645,8 +1646,7 @@ lUInt16 LVFreeTypeFace::measureText(const lChar32 *text,
                 cur_width += letter_spacing_w;
             }
             widths[i] = cur_width;
-            if ( !isHyphen ) // avoid soft hyphens inside text string
-                prev_width = cur_width;
+            prev_width = cur_width;
```

Comment:
Was this change voluntary ? Or is this taken care of elsewhere?
I think isHyphen maks soft-hyphens, that is a glyph that has a width. We don't draw them, so keeping their width in measureText() (as you're doing now here) might result in measureText() measuring words larger that what they will be when drawn.

Reply:
This is dead code.